cmd/podman/play_kube.go

```diff
@@ -282,7 +282,7 @@ func kubeContainerToCreateConfig(containerYAML v1.Container, runtime *libpod.Run
 		if err := validateVolumeCtrDir(volume.MountPath); err != nil {
 			return nil, errors.Wrapf(err, "error in parsing MountPath")
 		}
-		containerConfig.Volumes = append(containerConfig.Volumes, fmt.Sprintf("%s:%s", host_path, volume.MountPath))
+		containerConfig.Volumes = append(containerConfig.Volumes, fmt.Sprintf("%s:%s:z", host_path, volume.MountPath))
```

Comment:
Can we make appending `:z` conditional on SELinux being enabled?
@rhatdan thoughts?

Reply:
@mheon honestly I'm not even happy with this solution. The k8s api isn't expressive enough for what we need here (what if a user wants ro). I think I'm going to default to appending :z if the user doesn't specify in the path (<ctr_path>:z) in the yaml. Only way I can think to add the correct permission but also not mess with k8s api...